src/components/Menu.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+const RwdMenu = () => {
```

Comment:
Nit: component name should be the same as file name

Reply:
fix it commit : 902e2b